bin/resources/GameIndex.yaml

```diff
@@ -2509,6 +2531,8 @@ SCES-50781:
   name: "Destruction Derby Arena [Beta, Promo, & Full Retail]"
   region: "PAL-M6"
   compat: 5
+  gsHWFixes:
+    disableDepthSupport: 1 # Removes fog-like filter.
```

Comment:
Don't add these for DDS, this is a bug in the TC which is being fixed in #5349

Reply:
Oh okay